src/net/sf/memoranda/test/TestHour.java

```diff
@@ -0,0 +1,66 @@
+package net.sf.memoranda.test;
+
+import net.sf.memoranda.Event;
+import net.sf.memoranda.EventsManager;
+import net.sf.memoranda.EventsScheduler;
+import net.sf.memoranda.date.CalendarDate;
+import net.sf.memoranda.ui.App;
+import net.sf.memoranda.ui.EventDialog;
+import net.sf.memoranda.util.Local;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.Vector;
+
+import static org.junit.Assert.*;
+
+public class TestHour {
+
+    int repeatType, period, hour, minute;
+    String text;
+    CalendarDate startDate, endDate;
+    EventDialog dialog;
+    boolean workDays;
+
+    @BeforeClass
+    public static void setUpBeforeClass() throws Exception {
+    }
+
+    @Before
+    public void setUp() throws Exception {
+        repeatType = 5;
+        dialog = new EventDialog(App.getFrame(), Local.getString("New event"));
+        startDate = CalendarDate.today();
+        endDate = CalendarDate.tomorrow();
+        period = 1;
+        hour = 5;
+        minute = 12;
+        text = "hello";
+        workDays = dialog.workingDaysOnlyCB.isSelected();
+    }
+
+    @Test
```

Comment:
Please comment what you're tests do.  Doesn't have to be much but it's good practice because this is where most of the reviewing will be.